### PR TITLE
Configure top library order for homepage

### PR DESCRIPTION
**Pull request by dialelo**
Added a new configuration option: `exercises.top_libraries`, that will contain an array with the names of the libraries we want to show on top in the homepage. @raulraja @juanpedromoreno please take look!

### CODE REVIEW

File: site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala [hunk 1]
```diff
@@ -30,12 +33,15 @@ class ApplicationController(
     userProgressOps: UserProgressOps[ExercisesApp],
     T:               Transactor[Task]
 ) extends Controller with AuthenticationModule with ProdInterpreters {
+  implicit def application: Application = Play.current
+
+  lazy val topLibraries: List[String] = application.configuration.getStringList("exercises.top_libraries") map (_.asScala.toList) getOrElse Nil
```
Comment (juanpedromoreno) [inline site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala]: `application.configuration.getStringList("exercises.top_libraries").asScala` would work as well.

**Comment by juanpedromoreno** [comment]
Minor comment. LGTM.

File: site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala (hunk 1 above)
Comment (dialelo) [inline site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala]: Great suggestion, going to change it right away.

Comment (dialelo) [inline site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala]: Oops, didn't work. Take into account that `getStringList` returns an option.

Comment (juanpedromoreno) [inline site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala]: Uou! So the Play config seems to be different to the typesafe config library :/

https://github.com/typesafehub/config/blob/ba5e0da6941fd67b779d97c3bd82f1a9dfcfb657/config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala#L584

Hence LGTM!

**Pull request merged**